src/cljs/main/broadfcui/components/entity_details.cljs

```diff
@@ -32,7 +37,7 @@
            [:div {:style {:paddingTop "0.5rem"}}
             [:span {:style {:fontWeight 500 :marginRight "1rem"}} (if config? "Referenced Method:" "WDL:")]
             (links/create-internal {:onClick #(swap! state update :payload-expanded not)}
-              (if (:payload-expanded @state) "Collapse" "Expand"))
+                                   (if (:payload-expanded @state) "Collapse" "Expand"))
```

Comment:
Suspicious indentation…?

Reply:
Update to the newest `intellij-clojure-style.xml`